code/modules/paperwork/ticketmachine.dm

```diff
@@ -18,19 +18,23 @@
 	var/cooldown = 50
 	var/ready = TRUE
 	var/list/ticket_holders = list()
-	var/list/obj/item/ticket_machine_ticket/tickets = list()
+	var/list/tickets = list()
 	var/id = 1
 
+/obj/machinery/ticket_machine/Destroy()
+	ticket_holders.Cut()
+	..()
```

Comment:
we have to return the value or this won't work.

We need to qdel the tickets in this list.

Reply:
Fixed